bridgetown-website/frontend/styles/theme-dark.css

```diff
@@ -0,0 +1,93 @@
+:root.theme-dark {
+  --body-background: var(--color-darkest-gray);
+  --body-color: var(--color-light-gray);
+
+  --sl-color-primary-700: var(--color-light-green);
+  --sl-color-primary-600: var(--color-lighter-green);
+  --sl-color-primary-500: var(--color-lightest-green);
```

Comment:
A couple more lighter green colors were added here for functional reasons. 

Testing the site with Lighthouse in Chrome's devtools showed issues with color contrast for primary buttons against a dark background.

I expect these will need some design thought - they look out of sorts but I wasn't sure how else to approach this.

Example of these colors for a button and its hover state:

![image](https://user-images.githubusercontent.com/1370570/188709907-7b4cf337-ebc4-445e-a61a-4249cfbfd3f1.png)

![image](https://user-images.githubusercontent.com/1370570/188709940-e830630b-ae14-4ad2-a7d0-71f1d083200c.png)

Reply:
Hmm, yeah we'll need to think about the primary button style…either retain a dark green background with white text (perhaps with thin <=1px border around it), or create a more saturated green with black text. That particular example seems lackluster

Reply:
👍  Had a play around with these ideas..  let me know if you want me to push one of these as a commit.

![image](https://user-images.githubusercontent.com/1370570/192102176-4d20710b-15cb-4e92-808c-f06dddc2aa61.png)

![image](https://user-images.githubusercontent.com/1370570/192102266-97b1caa2-7919-4334-8949-52a3f53164f6.png)

Reply:
I like the dark with the outline (the first option)! In general it seems like dark modes which also offer better accessibility tend to increase outline contrast. 👍

Reply:
PR updated!